GNUmakefile

```diff
@@ -39,6 +39,10 @@ test:
 	cat coverage.out.tmp | grep -v "pb.go\|pb.validate.go\|generated.go\|indykite-sdk-go/test/\|main.go\|indykite-sdk-go/examples/" > coverage.out
 	rm coverage.out.tmp
 
+integration:
+	go clean -testcache && go test --tags=integration github.com/indykite/indykite-sdk-go/authorization
+	go clean -testcache && go test --tags=integration github.com/indykite/indykite-sdk-go/ingest
+
```

Comment:
1. Do we need `go clean -testcache`? 
2. Why you run separate packages and not just `go test --tags=integration ./...`?

Reply:
I added the go clean otherwise the tests are not run and just return "cached". Do you have better way to do it?
I will remove the separation

Reply:
OK, makes sense. I guess we want to run integration tests even they are not changed.

Thanks, remove separation